terraform/modules/gce/compute/nodejs/nodejs.sh.tpl

```diff
@@ -0,0 +1,278 @@
+#!/bin/bash
+
+NAME="${node_name}-$(hostname)"
+SANITIZEDNAME=$${NAME//-/_} # Replace hyphens with underscores, Consul Template doesn't like hyphens
+SSLCERTDIR=/usr/local/etc
+SSLSITECERTPATH=$SSLCERTDIR/site.crt
+SSLVAULTCERTPATH=$SSLCERTDIR/vault.crt
+NODEJSPOLICYNAME=${vault_policy}
+NODEJSPOLICY=/opt/vault/policies/$NODEJSPOLICYNAME.json
+GENERICSECRETPATH=secret/$NODEJSPOLICYNAME/$SANITIZEDNAME
+GENERICSECRETKEY=secret_key
+GENERICSECRET="This is a secret stored in Vault for $NAME using the $NODEJSPOLICYNAME policy"
+AWSROLEPOLICY=/opt/vault/policies/aws_$NODEJSPOLICYNAME.json
```

Comment:
We can probably remove the Vault AWS specific stuff here, it won't work in GCE and is overkill for this project. I would just remove all Vault stuff in general for now and we can rethink.